src/types/redis_zset.cc

```diff
@@ -744,16 +754,15 @@ rocksdb::Status ZSet::UnionStore(const Slice &dst, const std::vector<KeyWeight>
       }
     }
   }
-  if (!dst_zset.empty()) {
-    std::vector<MemberScore> mscores;
-    mscores.reserve(dst_zset.size());
+  auto compare_score = [](const MemberScore &score1, const MemberScore &score2) { return score1.score < score2.score; };
```

Comment:
@torwig / @git-hulk needed an opinion on this. If the scores of both the members are same during sorting, do we want to also lexicographically check the members as tie breakers or are we okay with them being in the order which they were originally present in?

Reply:
@uds5501 If I understand correctly, the initial order of how members are stored is already lexicographically sorted and retrieved in that order. @git-hulk Fix my thought if it is incorrect.

Reply:
> @uds5501 If I understand correctly, the initial order of how members are stored is already lexicographically sorted and retrieved in that order. @git-hulk Fix my thought if it is incorrect.

Yes, that's right.

Reply:
Understood, so I shold lexicographically sort the final result as well , right?

Reply:
Yes, exactly.

Reply:
Incorporated this sorting logic, thanks.